camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/common/KafkaReader.java

```diff
@@ -163,53 +168,96 @@ public boolean fetch() throws IOException {
     try {
       fetchResponse = simpleConsumer.fetch(fetchRequest);
       if (fetchResponse.hasError()) {
+        throw new RuntimeException();
```

Comment:
I think it's better to put the fetchResponse.error in the exception message so the caller will have more information to print.